apps/dashboard/src/pages/integrations/components/integration-card.tsx

```diff
@@ -0,0 +1,117 @@
+  const navigate = useNavigate();
+
+  const tableIntegration: ITableIntegration = {
+    integrationId: integration._id ?? '',
+    name: integration.name,
+    identifier: integration.identifier,
+    provider: provider.displayName,
+    channel: integration.channel,
+    environment: environment.name,
+    active: integration.active,
+  };
+
+  const handleConfigureClick = (e: React.MouseEvent) => {
+    if (integration.channel === 'in_app' && !integration.connected) {
+      e.stopPropagation();
+      navigate(ROUTES.INBOX_EMBED);
+    } else {
+      onRowClickCallback({ original: tableIntegration });
+    }
+  };
+
+  const isDemo = isDemoIntegration(provider.id);
+
+  return (
+    <div
+      className={cn(
+        'bg-card shadow-xs group relative flex min-h-[125px] cursor-pointer flex-col gap-2 overflow-hidden rounded-xl border border-neutral-100 p-3 transition-all hover:shadow-lg',
+        !integration.active && 'opacity-75 grayscale'
+      )}
+      onClick={handleConfigureClick}
+      data-test-id={`integration-${integration._id}-row`}
+    >
+      <div className="flex items-start justify-between">
+        <div className="flex items-center gap-1.5">
+          <div className="relative h-6 w-6">
+            <ProviderIcon
+              providerId={provider.id}
+              providerDisplayName={provider.displayName}
+              className="h-full w-full"
+            />
+          </div>
+          <span className="text-sm font-medium">{integration.name}</span>
+        </div>
+        {integration.primary && (
+          <Tooltip>
+            <TooltipTrigger>
+              <RiStarSmileLine className="text-feature h-4 w-4" />
+            </TooltipTrigger>
+            <TooltipContent>This is your primary integration for the {provider.channel} channel.</TooltipContent>
+          </Tooltip>
+        )}
+      </div>
+      <div className="flex items-center gap-2">
+        {isDemo && (
+          <Tooltip>
+            <TooltipTrigger className="flex h-[16px] items-center gap-1">
+              <span className="flex h-[16px] items-center gap-1">
+                <Badge variant="warning" className="text-2xs h-[16px] rounded-sm text-[#F6B51E]">
+                  DEMO
+                </Badge>
+              </span>
+            </TooltipTrigger>
+            <TooltipContent>
+              <p>
+                This is a demo provider for testing purposes only and capped at 300{' '}
+                {provider.channel === 'email' ? 'emails' : 'sms'} per month. Not suitable for production use.
+              </p>
+            </TooltipContent>
+          </Tooltip>
+        )}
+      </div>
+
+      <div className="mt-auto flex items-center gap-2">
+        {integration.channel === 'in_app' && !integration.connected ? (
+          <Button size="xs" className="h-[26px]" variant="outline" onClick={handleConfigureClick}>
```

Comment:
It's not a size that will be used outside of this component. @twentyone24 I've added this Configure button on the In App provider in case the user hasn't connected yet the integration. Any suggestions on other UX pattern for it? I had to adjust the button size to be in the same size of the Env pill

Reply:
my two cents:
- It doesn't look like a clickable button at glance, alongside the env pill. 
- since the whole card is clickable and does the same action to what clicking on configure does, I'd suggest to have a warning Icon to the the top-right instead. so, this adds an emphasis of action required / missing, and urge to configure. 

WDYT?

Reply:
@twentyone24 I'm not sure if it should be a warning. It's more of a Get Started state.